docs/src/features.md

```diff
@@ -272,15 +302,15 @@ This section quantifies the advantages of the [`flexible connection`](@ref flex-
 The table below shows the number of constraints and variables for each approach over a six-hour horizon. This highlights the potential of flexible time resolution in reducing the size of the optimization model.
 
 | Modeling approach                          | Nº Variables | Nº Constraints | Objective Function |
-|:-------------------------------------------|:------------ |:---------------|:------------------ |
+| :----------------------------------------- | :----------- | :------------- | :----------------- |
 | Classic approach with hourly resolution    | 48           | 84             | 28.4365            |
 | Flexible connection with hourly resolution | 42           | 72             | 28.4365            |
```

Comment:
Thanks, @gnawin. These numbers also need to be updated because they were calculated with the previous version of Tulipa. You can get the second row quickly by taking out the partitions on assets and flows. I'm afraid you need to add the new node for the first row, as in the example, which is not in the files. However, it is an excellent exercise to see how to work with Tulipa at the CSV level.

Reply:
Thanks! I'll do this.